source/gui/settingsDialogs.py

```diff
@@ -4388,11 +4388,20 @@ def makeSettings(self, settingsSizer):
 		outputsLabelText = _("&Output table:")
 		self.outTables = [table for table in tables if table.output]
 		self.outTableNames = [table.fileName for table in self.outTables]
-		outTableChoices = [table.displayName for table in self.outTables]
+		outTableForCurLangIndex = self.outTableNames.index(
+			brailleTables.getDefaultTableForCurLang(brailleTables.TableType.OUTPUT),
+		)
+		self.outTableForCurLang = self.outTables[outTableForCurLangIndex]
+		# Translators: An option in Braille settings to select a braille table automatically, according to the current language.
+		outTableChoices = [_("Automatic (%s)" % self.outTableForCurLang.displayName)]
```

Comment:
Please use modern string formatting
```suggestion
		outTableChoices = [_("Automatic ({name})").format(name=self.outTableForCurLang.displayName]
```